packages/components/src/shared/src/useId.ts

```diff
@@ -1,7 +1,23 @@
 import { isNil } from "./assertions";
-import { useId as useAutoId } from "@reach/auto-id";
+import { useId as useIdReact, useState } from "react";
+
+// Polyfill to support useId on React versions below 18.0.0, doesn't work on server
+let currentId = 0;
```

Comment:
wouldn't it be safer to still use the import { useId as useAutoId } from "@reach/auto-id"; in the meantime as the polyfill?

Reply:
As mentionned, i also wouldn't mind not having a polyfill and force all the clients to update to react 18. We rarely support older version of the apps

Reply:
I agree with Alex regarding not supporting app using React 17 and lower. Maybe it could work for useId but what about using other new features from React 18?

I think it's quite limiting and apps could afford using an older version of Orbit until they migrate to React 18 (which should not be a significant investment anyway)